.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,19 @@
+## What changes were proposed in this pull request?
+
+## Why are the changes needed?
+
+## Please briefly describe the changes such as API changes and code design.
+
+(This section is needed unless a github issue describing the design is provide below)
+
```

Comment:
Would it be possible to update the title **"Please briefly describe the changes such as API changes and code design."** to a more general and brief title such as **"Description"**?
```
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
```
Would it be better to put the `## Description` in the first section? And comment out the descriptive text using `<!--- just --->`